docker/brew.Dockerfile

```diff
@@ -107,8 +108,18 @@ RUN git config --global --add safe.directory ./
 RUN $YARN build --filter=backend
 
 # Build dynamic plugins
-# hadolint ignore=DL3059
-RUN $YARN --cwd ./dynamic-plugins export-dynamic
+RUN $YARN export-dynamic
+RUN $YARN clean-dynamic-sources
+RUN mkdir -p dynamic-plugins-root && \
+    cd dynamic-plugins-root && \
+    rm -Rf * && \
+    for pkg in $CONTAINER_SOURCE/dynamic-plugins/*/dist-dynamic; do \
+      if [ -d $pkg ]; then \
+        archive=$(npm pack $pkg) && \
+        tar -xzf "$archive" && rm "$archive" && \
```

Comment:
see above; you can do this a bit simpler if desired

Reply:
TBH, I'd rather go for the current way, so that I don't have to test it again :-)